src/Tickets/Emails/Email_Abstract.php

```diff
@@ -391,6 +381,260 @@ public function get_edit_url() {
 	 * @return string
 	 */
 	public function get_id(): string {
-		return static::$id;
+		return $this->id;
+	}
+
+	/**
+	 * Get setting option key.
+	 * 
+	 * @since TBD
+	 * 
+	 * @return string
+	 */
+	public function get_option_key( $option ): string {
+		$template_name = $this->template;
+		return "tec-tickets-emails-{$template_name}-{$option}";
```

Comment:
As mentioned above, the template name will refer to the file that's being called to generate the HTML. I like the idea of having something for that, maybe we need to define a different variable for it? Like a "prefix" or something. But we need to reserve the `$template` to be extended.

That way, if someone (we or third parties) want to have their own template for X email, they can do it.

Reply:
We'll use the new `slug` variable.